recipe/meta.yaml

```diff
@@ -1,46 +1,54 @@
-{% set version = "0.9.48" %}
-{% set name = "PyAutoGUI" %}
+{% set name = "pyautogui" %}
+{% set version = "0.9.53" %}
 
 package:
-  name: pyautogui
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: PyAutoGUI-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e91a25c1cdf826e7d0581775b5fbe47f7e12af79e0eb9dc3e1488ba99f2e0c60
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyAutoGUI-{{ version }}.tar.gz
+  sha256: d31de8f712218d90be7fc98091fce1a12a3e9196e0c814eb9afd73bb2ec97035
 
 build:
-  number: 2
+  skip: True  # [py>38]
+  number: 0
   script:
-    - "{{ PYTHON }} -m pip install . -vv"  # [not linux]
-    - DISPLAY=localhost:1.0 xvfb-run {{ PYTHON }} -m pip install . -vv  # [linux]
+    - {{ PYTHON }} -m pip install . -vv    # [not linux]
+    - DISPLAY=localhost: 1.0 xvfb-run {{ PYTHON }} -m pip install . -vv  # [linux]
```

Comment:
this is a dictionary. either quote the whole line, or remove the space

Reply:
got it